cyoa/app/views/users/index.html.haml

```diff
@@ -0,0 +1,24 @@
+- if @errors
+  .errors= @errors
+
+%h1 Leaderboard
+- @users.each_with_index do |user, i|
+  %p= "#{user.username} #{@hi_score_games[i].total_score}"
+
+- if logged_in?
+  ="WELCOME #{current_user.username}. TRY NOT TO SUCK TODAY"
```

Comment:
HAML can accept `#{}` escapes so you don't need `=""`

`WELCOME #{current_user.username}. TRY NOT TO SUCK TODAY`